src/dags/dag.py

```diff
@@ -183,21 +224,24 @@ def _create_combined_function_from_dag(
             Enforcing the signature has a small runtime overhead.
         set_annotations (bool): If True, we set the annotations of the concatenated
             function using the annotations of the functions that are used to generate
-            the targets. In case of a type mismatch, an error is raised.
+            the targets. In case of a type mismatch, an error is raised. Type
+            annotations must be strings, else a NonStringAnnotationError is raised.
+            Either enclose the annotations in quotes or add
```

Comment:
Thanks for adjusting! Is there a recipe for what to do if I don't have control over a function I want to include in the graph? Perfectly fine to leave out, just occurred to me.

Reply:
There is no built-in recipe in dags. You would probably have to set the `__annotation__` attribute of that function manually.

Reply:
Oh yes, this is what I meant. So a line like "if you do not have direct control over some function, you can wrap it in an annotated version" (